data/Page/social.md

```diff
@@ -53,7 +53,7 @@ All attendees, speakers, volunteers and organisers welcomed to our virtual dinne
 
 _When_: 7pm-9pm  
 
-_Where_: Zoom meeting, to be announced in venueless
+_Where_: Zoom meeting, to be announced in Venueless
 
 _Requirements_: A device with [zoom](https://zoom.us/download) installed
```

Comment:
Does Zoom's web client support breakout rooms these days?

Otherwise, just a tiny typo:
```suggestion
_Requirements_: A device with [Zoom](https://zoom.us/download) installed

Reply:
I believe webclients are now supported: https://support.zoom.us/hc/en-us/articles/115005769646-Participating-in-breakout-rooms

Reply:
would probably be good to also mention (or a web browser supported by Zoom) or similar then, but not a huge deal